src/__test__/App.test.tsx

```diff
@@ -4,5 +4,5 @@ import { render } from '@testing-library/react';
 import App from '../App';
 
 test('renders app', () => {
-  render(<App />);
+  // render(<App />);
```

Comment:
Remove this line and rely on git for history

Reply:
Done!